### ISSUE TITLE
Amend table caption

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/FreeBMD2#445]
@AlOneill noted: The table caption, heard by screen-reader users, is for example, "Information of Hackney District". To sound more natural, the "of" should be replaced with "about".

**Comment by Vino-S** [comment]
REady for testing in test3

**Comment by DeniseColbert** [comment]
@AlOneill could you check that this is OK (Vino says the code needs looking at)

**Comment by AlOneill** [comment]
The caption for Bakwell now reads:
"Information about Bakewell (1839 onw) District"

Thank you for using "about".

There is a problem with "1839 onw": I think that some of our users will struggle to understand what "onw" means. I suggest "from 1839". [This is a similar issue to the presentation of dates for sighted users, #450.]

**Comment by richpomfret** [comment]
"1839 and after" to be used instead. @Vino-S to make the change.

**Comment by Vino-S** [comment]
Ready for testing in test3

**Comment by richpomfret** [comment]
@AlOneill will you be able to take a look at/test this one?

**Comment by PatReynolds** [comment]
@DeniseColbert to test.

**Comment by DeniseColbert** [comment]
Just read the story properly, sorry, and this needs a screen reader to test so have reassigned @AlOneill

**Comment by PatReynolds** [comment]
Pat to test (using screen reader).

**Comment by AlOneill** [comment]
@PatReynolds 

On an **Entry Information** page, VoiceOver tells me I'm in a table and then says, for example:
"Entry Information of Thomas O'Neill"

I thought we had agreed to make the wording of this `caption` more natural? But perhaps this was never discussed or is part of another issue!

On a **District information** page, I hear:
Information about Bakewell 1 8 3 9 and after District

However, a S-R user might also hear the `h1` heading which VO reads as:
District Information Bakewell 1 8 3 9 O N W

… which is very similar, but not as clear.
Note that "onw" is read as "O N W", with no attempt to expand it and I hear the year as 4 separate digits.

Once @PatReynolds has tested with a Windows screen-reader, I think we need to decide on how to make such headings clearer. I will investigate whether VO can read the year in a more helpful manner.

**Comment by PatReynolds** [comment]
Pat to re-record as sound quality not good enough.

**Comment by richpomfret** [comment]
To also go with Vino's suggestion and just have 'Bakewell'.

**Comment by Vino-S** [comment]
Ready for testing in test3

**Comment by PatReynolds** [comment]
@PatReynolds

**Comment by PatReynolds** [comment]
On NvDA I now hear "Bakewell 1839 on",  in the table which is fine (not sure why 'information about was needed.  

District detail page reads out "District information Bakewell" (the text) - again seems fine.

**Comment by AlOneill** [comment]
Table caption looks fine on t3 but on beta I got "We're sorry, but something went wrong with processing your request …" when I selected the "View" button in the listing (table) of Districts beginning with B.

**Comment by DeniseColbert** [comment]
@Vino-S to update  (ref. 1993 records)

**Comment by DeniseColbert** [comment]
@Vino-S to take a look

**Comment by DeniseColbert** [comment]
I can access B districts (well, Bala) so is this fixed?

**Comment by DeniseColbert** [comment]
@AlOneill could you please take a look at Beta to see if this is fixed?

**Comment by AlOneill** [comment]
@DeniseColbert 
Issue appears to be fixed on beta. Caption is good.

**Comment by DeniseColbert** [comment]
Done, closing
